addon/helpers/current-quarter.js

```diff
@@ -1,8 +1,23 @@
 import { helper } from "@ember/component/helper";
 import { DateTime } from "luxon";
 
-const currentQuarter = function isUnset([format = "q/yyyy"]) {
-  return `Q${DateTime.now().toFormat(format)}`;
+// The quarterly closure is always two weeks before and after the finish of the quarter.
```

Comment:
I would extend it until at least 1-2 months after the finish of the quarter :sweat_smile:

Reply:
Done